validate_sample.py

```diff
@@ -56,6 +56,7 @@ def __init__(self):
             'FIELDOFFICER_ID': [max_length(10)],
             'TREATMENT_CODE': [mandatory(), in_set(self.TREATMENT_CODES)],
             'CE_EXPECTED_CAPACITY': [numeric(), max_length(4)],
+            'CE_SECURE': [mandatory(), max_length(2)]
```

Comment:
Should this be 1 or 0 be max_length(2)?

Reply:
In could be `in_set({'0','1'})` for maximum safety